wfe/wfe.go

```diff
@@ -836,6 +836,12 @@ func (wfe *WebFrontEndImpl) updateChallenge(
 				fmt.Sprintf("Failed to validate challenge: %q", err)), response)
 		return
 	}
+	response.Header().Add("Link", link(existingChal.Authz.URL, "up"))
```

Comment:
Aha! I forgot about these relations. I think the Authz's need an "up" link to their order as well. Do you want to add that into this PR or should I carry it into my WIP?